scripts/test-captplanet.sh

```diff
@@ -13,30 +13,36 @@ kill -9 $CAPT_PID
 captplanet run &
 CAPT_PID=$!
 
+#setup tmpdir for testfiles and cleanup
+TMP_DIR=/tmp/capt
```

Comment:
Depending on the system the temp directory might be somewhere else. Although probably won’t cause too much problems.

Reply:
We should switch depending on the OS to another path

Reply:
I thought about this, but I didn't know if we wanted to try and support windows. It's pretty easy to change this to be just the current directory.

Reply:
if you use `$(mktemp -d)` you should get a cross-platform tmp directory. you can also set an exit trap to clean up the directory after the test ends http://redsymbol.net/articles/bash-exit-traps/

Reply:
so it would be `TMP_DIR=$(mktemp -d) like that?

Reply:
Updated the script to use `mktemp` for both the tmp and cmp directories